1-js/02-first-steps/11-logical-operators/4-alert-and/solution.md

```diff
@@ -1,10 +1,11 @@
 The answer: `1`, and then `undefined`.
```

Comment:
The answer: `1`, and then `undefined`. 

এই লাইনটি রিমুভ করুন এবং এখান থেকে লাইন নাম্বার গুলো ঠিক করুন।